templates/stamp/mysql-ubuntu-install.sh

```diff
@@ -0,0 +1,380 @@
+#!/bin/bash
+
+# general parameters
+PACKAGE_VERSION=5.7
+PACKAGE_NAME=mysql-server
+MYSQL_SERVER_PACKAGE_NAME="${PACKAGE_NAME}-${PACKAGE_VERSION}"
+
+MYSQL_REPLICATION_NODEID=
+NODE_ADDRESS=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'`
+
+NETWORK_PREFIX="10.0.0"
+MASTER_NODE_IPADDRESS=
+MYSQL_REPLICATION_USER=lexoxamysqlrepluser
+MYSQL_REPLICATION_PASSWORD=f@ncyP@ssW0rd!
+MYSQL_ADMIN_USER=lexoxamysqladmin
+MYSQL_ADMIN_PASSWORD=f@ncyP@ssW0rd!
+MYSQL_PORT=3306
```

Comment:
I assume we'd always override the password defaults. Why not just remove and fail if not provided?